mtp_api/apps/security/urls.py

```diff
@@ -35,6 +35,9 @@
 
 router.register(r'searches', views.SavedSearchView)
 
+router.register(r'security/checks', views.CheckView, basename='security-check')
```

Comment:
@ushkarev do you like `security/checks` or should we use `credits/checks` instead?

Reply:
i prefer `security/checks` in case other models need to be included in the future